mappings/net/minecraft/util/Clearable.mapping

```diff
@@ -1,3 +1,4 @@
 CLASS net/minecraft/class_3829 net/minecraft/util/Clearable
 	METHOD method_16825 clear (Ljava/lang/Object;)V
+		ARG 0 o
```

Comment:
What is this? What's passed as an argument when `clear` is called?

Reply:
I think is is like a static method that does
```java
static void clear(Object o) {
  if (o instanceof Clearable) ((Clearable) o).clear();
}
```

Reply:
Oh. If it's static then `o` is OK.

Reply:
Yeah, it's static.